packages/host/app/services/matrix-service.ts

```diff
@@ -74,6 +78,10 @@ const AI_BOT_POWER_LEVEL = 50; // this is required to set the room name
 const DEFAULT_PAGE_SIZE = 50;
 const MAX_CARD_SIZE_KB = 60;
 
+const DefaultSkillCards = [
+  `${baseRealm.url}SkillCard/43f62768-cbdb-401a-8130-30e12a29cf0d`,
```

Comment:
My only main note is that it might be nice to have this named so we have either in this file or elsewhere something like

`BOXEL_EDITING_SKILL = ...`

and then default skills as a list is more readable. Buck also mentioned renaming the skills cards themselves so that make make this less relevant as SkillCard/card-editing would be more obvious